utils/providerAPI/getUserRepositories.ts

```diff
@@ -63,6 +140,10 @@ export const getUserRepositories = async (session: Session) => {
 						const userReposPromiseGitHub = getUserRepositoriesForGitHub(access_key, authId)
 						allUserReposPromises.push(userReposPromiseGitHub);
 						break;
+					case 'bitbucket':
```

Comment:
low logaf - enum instead of string?

Reply:
JavaScript actually does not have natively implemented enums. The way we implement enums in JS is using `Object.freeze()`. Here, however, I don't think we need that because I have implemented a TS type that restricts the variable `repoProvider` to only be one of `github`, `bitbucket` and `gitlab`. This is so because the variable `supportedProviders` is of the `RepoProvider` type.

Reply:
Lol, really? JS doesn't have enums? 😆